src/logical/model.rs

```diff
@@ -34,6 +35,12 @@ impl Identifier {
     }
 }
 
+impl std::fmt::Display for Identifier {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        std::fmt::Display::fmt(&self.name(), f)
```

Comment:
I think the common way of implementing this is:
```suggestion
        write!(f, "{}", self.name())
```
But maybe I'm wrong; I'm fine with both.